solaris/src/tests/mod.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+mod multisig;
```

Comment:
Absolutely, wasn't sure how to structure the tests. Will all the contract tests go into examples, or will they be tested in-place, e.g. the `contracts` repo?

Reply:
I think the tests should be part of `contracts` repo, this one should contain only a library (`solaris`) and couple of examples so that people can start writing their own tests easier.